povm_toolbox/quantum_info/base_frame.py

```diff
@@ -0,0 +1,75 @@
+# (C) Copyright IBM 2024.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""TODO."""
+
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import Generic, TypeVar
+
+import numpy as np
+from qiskit.quantum_info import Operator, SparsePauliOp
+
+# type of the labels used to specify frame operators
+LabelT = TypeVar("LabelT")
```

Comment:
It should not be necessary to duplicate this.
Actually, the right place is probably here and `BaseDUAL` should not have another version of it. Either imported it from here or leave it generic without passing `LabelT` into `BaseFrame` again